io/plugins/eu.esdihumboldt.hale.io.appschema/src/eu/esdihumboldt/hale/io/geoserver/Namespace.java

```diff
@@ -86,4 +103,43 @@ protected String templateLocation() {
 		return TEMPLATE_LOCATION;
 	}
 
+	public static Namespace fromDocument(Document doc) {
```

Comment:
Missing JavaDoc for `public` declaration

Reply:
Fixed.